sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py

```diff
@@ -28,10 +28,11 @@
     _get_endpoint_only_fqdn,
     _get_authentication_policy,
     _is_cloud_event,
+    _is_eventgrid_event,
```

Comment:
same questions as the sync publisher client

Reply:
similar design as sync